cmd/test-external-check/README.md

```diff
@@ -27,12 +29,14 @@ spec:
       - khc
 ```
 
+##### Example Check Template
+
+The pod is designed to run continuously and wait every few seconds then report a success or failure when communicating to the Kuberhealthy Servers. The pod is also design to log any errors it encounters when communicating to the KuberHealthy API.
```

Comment:
sorry nitpicky! Kuberhealthy -- H is lowercased :)

Reply:
haha! ok